tests/unit/sparse_attention/research_attention/maskers/fixed/implementations/test_double_sparsity_correctness.py

```diff
@@ -0,0 +1,415 @@
+"""
```

Comment:
do we want author section? would remove otherwise in favor of less code. nit: dates look wrong